cannon/multicannon/witness.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"os"
+
+	"github.com/ethereum-optimism/optimism/cannon/cmd"
+	"github.com/urfave/cli/v2"
+)
+
+func Witness(ctx *cli.Context) error {
+	version, err := parseVersionFlag(ctx.String(WitnessVersionFlag.Name))
```

Comment:
Can we not detect the version from the "input" flag here like we do in `run.go`?

Reply:
Indeed. Wil do this.